client/js/lounge.js

```diff
@@ -164,7 +164,7 @@ $(function() {
 		const target = self.attr("data-target");
 
 		if (!target) {
-			return false;
+			return;
```

Comment:
This is the actual bugfix. 

As it turns out `button` is `clicked` even when using enter keypress, unlike `input type="button"` so this click handler prevented it from being submitted.